src/Heading.js

```diff
@@ -19,5 +28,40 @@ Heading.propTypes = {
   ...COMMON.propTypes,
   ...TYPOGRAPHY.propTypes
 }
+//
+
+const HeadingInternal = styled.h2`
```

Comment:
This is basically the same as `Heading` but without a default font size. All of the stuff above this line will get removed when we deprecate `Heading`